superset/assets/javascripts/dashboard/v2/containers/DashboardComponent.jsx

```diff
@@ -51,6 +51,11 @@ function mapStateToProps({ dashboardLayout: undoableLayout, editMode }, ownProps
         );
       }
     });
+  } else if (props.component.type === CHART_TYPE) {
+    const chartKey = props.component.meta && props.component.meta.chartKey;
```

Comment:
why are you calling this `key` whereas everything else is an `id`? can you use `id` instead?